pkgs/applications/networking/utahfs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, lib, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "utahfs";
+  version = "1.0";
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1hpwch5fsqlxwpk5afawa1k5s0bx5c1cw0hvdllp7257lgly19fb";
+  };
+
+  goPackagePath = "github.com/cloudflare/utahfs";
+  goDeps = ./deps.nix;
+
+  meta = with lib; {
+    homepage = "https://github.com/cloudflare/utahfs";
+    description =
+      "Encrypted storage system that provides a user-friendly FUSE drive backed by cloud storage";
+    license = licenses.bsd3;
+    maintainers = [ maintainers.snglth ];
```

Comment:
Nit: I see `with maintainers; [ snglth ];` used more frequently, but no big deal.

Reply:
It's really pointless if there is only one maintainer or whatever to be put into expression's lexical scope, IMHO.